meson.build

```diff
@@ -0,0 +1,260 @@
+# for ninja:
+# meson setup builddir
+# cd builddir ; meson compile
+
+# fox Xcode:
+# meson setup --backend=xcode xcode
+# cd xcode ; meson compile
+
+# regenerate configuration (run from builddir/)
+# meson setup --reconfigure --wipe
+
+project('pcsc-lite', 'c',
+  version : '2.2.0')
+
+# for config.h
+conf_data = configuration_data({
+  'VERSION' : '"' + meson.project_version() + '"',
+  'PCSCLITE_HP_DROPDIR' : '"' + get_option('usbdropdir') + '"',
+  'HAVE_NANOSLEEP' : true,
+  })
+# for generated files from .in templates
+confgen_data = configuration_data({
+  'VERSION' : meson.project_version(),
+  'ipcdir' : get_option('ipcdir'),
+  'sysconfdir' : get_option('sysconfdir'),
+  'sbindir_exp' : get_option('sbindir'),
+  'PCSCLITE_CONFIG_DIR' : get_option('serialconfdir'),
+  'usbdropdir' : get_option('usbdropdir'),
+  })
+features = []
```

Comment:
nit: I would move this below the compiler checks under the `# tests for options` as it is the first place accessing this array.

Reply:
done